### PR TITLE
Tests pass windows

### PR DESCRIPTION
**Pull request by AustinArey**
See #14

### CODE REVIEW

File: path/globRoot.ts [hunk 1]
```diff
@@ -1,10 +1,12 @@
-import { isGlob } from "../_deps/path.ts";
+import { isGlob, sep } from "../_deps/path.ts";
+
+/** This code should work for both Windows and Mac systems, as the sep variable will be set to the appropriate separator character based on the current operating system. The dirname function will also handle any differences in how file paths are formatted between Windows and Mac. */
```
Comment (pskfyi) [inline path/globRoot.ts:3]: Can we remove this comment?

File: .github/workflows/pr-checks.yml [hunk 2]
```diff
@@ -24,22 +24,22 @@ jobs:
           deno-version: v1.x
 
       - name: Lint
-        # Intentionally only on Linux because output will be the same on all 
+        # Intentionally only on Linux because output will be the same on all
```
Comment (pskfyi) [inline .github/workflows/pr-checks.yml:27]: I can't tell why these lines are updated. Is their line ending changed to `\r\n`?

**Review by pskfyi — COMMENTED** [review]
Looking fantastic! 👏 

Just a few comments. Feel free to push back if you disagree.

The commit history is gorgeous 🤩  My only note is that the commit "fix(fs/glob): moved imports to from "./_deps" looks like it can be squashed into another commit, but if it does deserve to be its own commit then it seems to have a typo.

File: .github/workflows/pr-checks.yml (hunk 2 above)
Comment (AustinArey) [reply .github/workflows/pr-checks.yml:27]: Weird I did a regex search and only see `\n` here. I don't remember accidentally touching this line

**Comment by AustinArey** [comment]
> Looking fantastic! 👏
> 
> Just a few comments. Feel free to push back if you disagree.
> 
> The commit history is gorgeous 🤩 My only note is that the commit "fix(fs/glob): moved imports to from "./_deps" looks like it can be squashed into another commit, but if it does deserve to be its own commit then it seems to have a typo.

After the fixup I have suggestion to Sync 7down 6up. I believe I need to simply `git push -f` to replace this branches git history with my revised history. Correct?

**Review by pskfyi — APPROVED** [review]
Tremendous!!

**Pull request merged**